codegra_fs/constants.py

```diff
@@ -22,8 +22,40 @@
 url_help = """The url to find the api. This defaults to
 'https://codegra.de/api/v1/'. This is most likely
 'https://{UNIVERSITY}.codegra.de/api/v1/' where '{UNIVERSITY}' is your
-university (for example 'uva' for the University of Amsterdam).  This value can
-also be passed as an environment variable 'CGAPI_BASE_URL'"""
+university (for example 'uva' for the University of Amsterdam and 'ut' for the
+University of Twente).  This value can also be passed as an environment
+variable 'CGAPI_BASE_URL'"""
 
 all_submissions_help = """See all submissions not just the latest submissions
 of students."""
+
+cgfs_epilog="""possible environment variables
+
+
+ CGAPI_BASE_URL         Used as the value for the `--url` flag. It can be very
+                        convenient to set this environment variable in your
+                        `.bashrc` file (or equivalent).
+ CGFS_PASSWORD          Used as your password. Please take care when using this
+                        environment variable as all processes can read
+                        environment variables meaning this could easily leak
+                        your password. We recommend piping your password, see
+                        the last example for how to do this.
+
+examples:
+  cgfs -u https://uva.codegra.de/api/v1/ -m user ~/mount
+                        Mount the file system from https://uva.codegra.de/
+                        showing only your assignments in the directory
+                        "$HOME/mount".  You will be prompted for a password.
+  cgfs -u https://uva.codegra.de/api/v1/ -fm user ~/mount
+                        Same as above but mount in fixed mode, which means you
+                        cannot change or add files. This is useful for
```

Comment:
You can still add files in fixed mode, right? At least that's what it says in the help of the `--fixed` argument. Or can you still add files but they aren't uploaded? Then it should mention that in the `--fixed` description.

Reply:
You can add files but they won't be uploaded. I think this should be fixed in the `--fixed` explanation.